code.js

```diff
@@ -1,3 +1,18 @@
 function augmentingPath(graph, start, end) {
-    return [];
+    graph[start].v = "v"
+    const ke = Object.keys(graph[start])
+    if(start == end){
+        return [end];
+    }
+    for(n = 0; n < ke.length-1; n++){
+        if(graph[ke[n]]['v'] != "v"){
+            a = [start].concat(augmentingPath(graph, ke[n], end))
+            if(a[a.length-1] != -1){
+                return a}
+            a.pop()
+        }
+    }
+    n--
+    n--
+    return [-1];
```

Comment:
That's not what you're supposed to return when there's no path.

Reply:
Still relevant.

Reply:
it no longer returns -1 when the code finishes running

Reply:
Then why is the `return [-1];` statement still there?